packages/DataGrid/src/DataGrid.styles.js

```diff
@@ -0,0 +1,258 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+const experimentalGrey = "#e7e7e7"; /* experimental */
+export const Grid = styled.div.attrs(({ $width, isVisible }) => {
+  const _isVisible = isVisible ? { opacity: 0 } : { opacity: 100 };
+
+  return {
+    style: { width: `${$width}px`, ..._isVisible },
+  };
+})`
+  * {
+    box-sizing: border-box;
+  }
+  .grid--is-active {
+    box-shadow: ${tokens.highlight.active.withBorder.insetBoxShadow};
+  }
+
+  .grid--is-blur {
+    box-shadow: 0 0 0;
+  }
+
+  [role="gridcell"] {
+    /* this hide the aria "voice over" focus message */
+    &:focus {
+      outline: 0;
+    }
+  }
+
+  border-top: 1px solid ${tokens.border.color};
+  background: {tokens.color.white};
+
+  position: relative;
+
+  ${({ gridId }) => {
+    return css`
+      .grid-${gridId} {
+        overflow: scroll !important;
+      }
+
+      .${gridId}-header {
+        overflow: hidden !important;
+      }
+
+      .${gridId}-sticky-columns {
+        /*
+          The arrow navigatoion is border sensible, please avoid adding a border to this styled components
+          doing it will create a incorrect navigation while using the arrow keys.
```

Comment:
The arrow navigation is sensitive with borders. If a border is used, it will cause issues keyboard navigation.